src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/genericwhitespace/InputGenericWhitespaceDefault.java

```diff
@@ -19,8 +19,8 @@ void meth()
     {
         List<Integer> x = new ArrayList<Integer>();
         List<List<Integer>> y = new ArrayList<List<Integer>>();
-        List < Integer > a = new ArrayList < Integer > (); // 6 violations
-        List < List < Integer > > b = new ArrayList < List < Integer > > (); // 14 violations
+        List < Integer > a = new ArrayList < Integer > (); // 7 violations
+        List < List < Integer > > b = new ArrayList < List < Integer > > (); // 15 violations
     }
```

Comment:
Please add to input code with double `>>` like https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/28c7818_2024125529/reports/diff/elasticsearch/index.html#A1
To have single violation on it.

And add https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/28c7818_2024125529/reports/diff/Hbase/index.html#A1

And https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/28c7818_2024125529/reports/diff/Hbase/index.html#A3

Reply:
[Done](https://github.com/checkstyle/checkstyle/blob/0eaeb24626e64991d308fa5739cf8decc18d9b2b/src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/genericwhitespace/InputGenericWhitespaceBeforeCtorInvocation.java#L31-L41)

Reply:
>283     this.regionsToReopen = Collections.synchronizedMap
284                            (new HashMap<String, HRegionInfo> ());

I did not find in `(new ....` in your Inputs.

Reply:
> I did not find in (new .... in your Inputs. please add

https://github.com/checkstyle/checkstyle/blob/0eaeb24626e64991d308fa5739cf8decc18d9b2b/src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/genericwhitespace/InputGenericWhitespaceBeforeCtorInvocation.java#L39